wire-ios-data-model/Source/MLS/MLSService.swift

```diff
@@ -518,42 +523,35 @@ public final class MLSService: MLSServiceInterface {
         }
     }
 
-    private func claimKeyPackages(for users: [MLSUser]) async throws -> [KeyPackage] {
-        logger.info("claiming key packages for users: \(users)")
-        do {
-            guard let context = context else { return [] }
+    private func claimKeyPackages(
+        for users: [MLSUser]
+    ) async throws -> [KeyPackage] {
 
-            var result = [KeyPackage]()
+        guard let context = context else { return [] }
```

Comment:
**suggestion**: to be consistent with other places, we might want to add:
`assertionFailure("MLSService.context is nil")`